server/repository/assessment/getCountryStatus.ts

```diff
@@ -13,7 +14,7 @@ export const getCountryStatus = async (
 
   return client.oneOrNone<CountryStatus>(
     `
-          select * from ${assessmentCycleName}.country_status where country_iso = $1
+          select country_iso, props->>'status' as status, (props->>'desk_study')::boolean as desk_study from ${assessmentCycleName}.country where country_iso = $1
```

Comment:
use schemes to get schema name. pass assessment and cycle from apis

Reply:
Deprecated file